hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -90,10 +91,16 @@ public void run() {
     public static void main(final String... args) throws Exception {
         BootstrapUtils.setupConstructableRegistry();
         final var registry = ConstructableRegistry.getInstance();
-        new SwirldsPlatformBuilder()
-                .withName("Hedera")
-                .withMain(() -> new Hedera(registry))
-                .withNodeId(args != null && args.length > 0 ? Integer.parseInt(args[0]) : 0)
-                .buildAndStart();
+
+        final Hedera hedera = new Hedera(registry);
+        final NodeId selfId = args != null && args.length > 0 ? new NodeId(Integer.parseInt(args[0])) : new NodeId(0);
+
+        final PlatformBuilder builder = new PlatformBuilder(
+                Hedera.APP_NAME, Hedera.SWIRLD_NAME, hedera.getSoftwareVersion(), hedera::newState, selfId);
+
+        final SwirldsPlatform platform = builder.build();
+        hedera.init(platform, selfId);
+        platform.start();
+        hedera.run();
```

Comment:
I still hate this code block. It is way too complicated. Either hedera.run() or platform.start() should call the other 2 methods. No need to do this in the PR.